<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MessageIX Data Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="leftPanel">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="leftSplitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QWidget" name="navigator"/>
          <widget class="QTreeWidget" name="param_tree">
           <property name="headerLabel">
            <string>Parameters</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="contentWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QSplitter" name="contentSplitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QWidget" name="dataContainer">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QSplitter" name="dataSplitter">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <widget class="QWidget" name="tableContainer">
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <layout class="QHBoxLayout" name="titleLayout">
                  <item>
                   <widget class="QLabel" name="param_title">
                    <property name="text">
                     <string>Select a parameter to view data</string>
                    </property>
                    <property name="styleSheet">
                     <string>font-size: 12px; color: #333; padding: 5px; background-color: #f0f0f0;</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="view_toggle_button">
                    <property name="text">
                     <string>Raw Display</string>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                    <property name="styleSheet">
                     <string>font-size: 11px; padding: 3px 8px; background-color: #e0e0e0; border: 1px solid #ccc; border-radius: 3px;</string>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>120</width>
                      <height>0</height>
                     </size>
                    </property>
                   <property name="maximumSize">
                     <size>
                      <width>120</width>
                      <height>30</height>
                     </size>
                    </property>
                    </widget>
                  </item>
                  <item>
                   <widget class="QGroupBox" name="selector_container">
                    <property name="title">
                     <string>Data Filters</string>
                    </property>
                    <property name="visible">
                     <bool>false</bool>
                    </property>
                    <property name="styleSheet">
                     <string>font-size: 11px; font-weight: bold; margin-top: 0px; margin-left: 10px; padding-top: 5px; border: 1px solid #ccc; border-radius: 3px;</string>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                     <property name="leftMargin">
                      <number>10</number>
                     </property>
                     <property name="topMargin">
                      <number>5</number>
                     </property>
                     <property name="rightMargin">
                      <number>10</number>
                     </property>
                     <property name="bottomMargin">
                      <number>5</number>
                     </property>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QTableWidget" name="param_table">
                  <property name="alternatingRowColors">
                   <bool>true</bool>
                  </property>
                  <property name="styleSheet">
                   <string>QHeaderView::section { background-color: #e0e0e0; padding: 4px; border: 1px solid #ccc; font-weight: bold; font-size: 12px; }</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="graphContainer">
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <layout class="QHBoxLayout" name="chartControlsLayout">
                  <item>
                   <widget class="QPushButton" name="simple_bar_btn">
                    <property name="text">
                     <string>Simple Bar</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="stacked_bar_btn">
                    <property name="text">
                     <string>Stacked Bar</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="line_chart_btn">
                    <property name="text">
                     <string>Line Chart</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="stacked_area_btn">
                    <property name="text">
                     <string>Stacked Area</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="chartControlsSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QWebEngineView" name="param_chart">
                  <property name="minimumSize">
                   <size>
                    <width>300</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QTextEdit" name="console">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>300</height>
            </size>
           </property>
           <property name="plainText">
            <string>Welcome to MessageIX Data Manager
            </string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Input_File"/>
    <addaction name="actionOpen_Results_File"/>
    <widget class="QMenu" name="menuFile_separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <widget class="QMenu" name="menuFile_separator_2"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionRun_Solver"/>
    <addaction name="actionStop_Solver"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionDashboard"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRun"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <widget class="QProgressBar" name="progress_bar">
    <property name="visible">
     <bool>false</bool>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>0</height>
     </size>
    </property>
   </widget>
  </widget>
  <action name="actionOpen_Input_File">
   <property name="text">
    <string>Open Input File</string>
   </property>
  </action>
  <action name="actionOpen_Results_File">
   <property name="text">
    <string>Open Results File</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionRun_Solver">
   <property name="text">
    <string>Run Solver</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionStop_Solver">
   <property name="text">
    <string>Stop Solver</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionDashboard">
   <property name="text">
    <string>Dashboard</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>PyQt5.QtWebEngineWidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
